### DISCUSSION TITLE
What is this and how to remove this?

### DISCUSSION THREAD

**Discussion by httpanand** [community/community#15436] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/87514488/164874463-62830dec-f422-4f8d-8867-56fa9915d82e.png)
Please add a feature to remove this or make it available only for new users :/

**Comment by chase** [comment]
I agree, this should really only be for new accounts.

I've been using Github for quite a long time (since 2008), but it advises me to use basic features as if I've never used it before.

**Comment by httpanand** [comment, marked as answer]
Ok thank You  @github ...
